#!/usr/bin/python3


def raise_exception_msg(message=""):
    """A Function used to Raise a NameError exception with a message."""

    raise NameError(message)
